public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -79,6 +79,49 @@ const useIsNewGroup = (folder: string, group: string) => {
   return !groupIsInGroupOptions(group);
 };
 
+export const EvaluateEveryNewGroup = ({ rules }: { rules: RulerRulesConfigDTO | null | undefined }) => {
+  const {
+    watch,
+    register,
+    formState: { errors },
+  } = useFormContext<RuleFormValues>();
+  const styles = useStyles2(getStyles);
+  const evaluateEveryId = 'eval-every-input';
+  return (
+    <Field
+      label="Evaluation interval"
+      description="Evaluation interval applies to every rule within a group. It can overwrite the interval of an existing alert rule."
```

Comment:
Do we need this description here if we're creating a new group? It might confuse users a bit perhaps in this case?

Reply:
I was really confused about how to improve this part. And I finally decided to reduce some redundancy but not removing the description for the interval as we let the user to edit it in here, and we know it's not a clear concept.
<img width="543" alt="Screenshot 2022-12-14 at 17 14 42" src="https://user-images.githubusercontent.com/33540275/207651317-3896f80b-9680-420a-be02-61ffc2c9a511.png">